src/applications/vre/28-8832/config/chapters/status-selection/wizard.js

```diff
@@ -0,0 +1,20 @@
+export const schema = {
+  type: 'object',
+  properties: {
+    status: {
+      type: 'string',
+      enum: [
+        'Active duty service member',
+        'Veteran',
+        'Spouse of a Veteran or service member',
+        'Child of a Veteran or service member',
+      ],
```

Comment:
If possible, it'd be better if these were values that don't contain white space. Something like an enum with the values `['isActiveDuty', 'isVeteran', 'isSpouse', 'isChild']`, and then use enumTitles to describe the UI.